src/main/scala/PcapExample.scala

```diff
@@ -69,14 +63,16 @@ object PcapExample {
         !(data + offsetBytes + n)
       }
       .foreach { v =>
-        native.stdio.printf(toCString("%02X"), v)
+        native.stdio.printf(c"%02X", v)
       }
     println("...]")
   }
 
-  def main(args: Array[String]): Unit = {
+  def main(args: Array[String]): Unit = Zone { implicit zone => run(args) }
+
+  def run(args: Array[String])(implicit zone: Zone): Unit = {
     val cooked = args.contains("cooked")
-    val live = args.contains("live")
+    val live = !cooked
```

Comment:
live & cooked are not related. Live/non-live = read from network/non-network; cooked/non-cooked = read from a linux SSL input, whether file or not.

Did the article give the impression that they are linked? I'd like to fix that.